docs/docs/legal/data-processing-agreement.md

```diff
@@ -0,0 +1,229 @@
+<table>
+  <tr>
+    <th>Type of personal data</th>
+    <th>Examples</th>
+    <th>Affected data subjects</th>
+  </tr>
+  <tr>
+    <td><strong>Basic data</strong></td>
+    <td>
+      <ul>
+        <li>Surname and first name</li>
+        <li>Email addresses</li>
+        <li>User name</li>
+        <li>Language</li>
+      </ul>
+    </td>
+    <td>All users</td>
+  </tr>
+  <tr>
+    <td><strong>Login data</strong></td>
+    <td>
+      <ul>
+        <li>Randomly generated ID</li>
+        <li>Password</li>
+        <li>Public keys / certificates (&quot;FIDO2&quot;, &quot;U2F&quot;, &quot;x509&quot;, ...)</li>
+        <li>User names or identifiers of external login providers</li>
+        <li>Phone number(s)</li>
+      </ul>
+    </td>
+    <td>
+      <p>All users</p>
+      <p>Password: Users who use authentication methods with password.</p>
+      <p>Public Keys: Users who use an authentication procedure with cryptographic keys.</p>
+      <p> External login provider identifiers: Users who use an external login provider.</p>
+      <p>Phone number: Users who use authentication methods with SMS</p>
+    </td>
+  </tr>
+  <tr>
+    <td><strong>Profile data</strong></td>
+    <td>
+      <ul>
+        <li>Profile pictures</li>
+        <li>Gender</li>
+        <li>Birthday</li>
+        <li>Language</li>
+        <li>Address(es)</li>
+        <li>Phone number(s)</li>
+      </ul>
+    </td>
+    <td>Users who voluntarily add profile data</td>
+  </tr>
+  <tr>
+    <td><strong>Communication data</strong></td>
+    <td>
+      <ul>
+        <li>Emails</li>
+        <li>Chats</li>
+        <li>Call metadata</li>
+      </ul>
+    </td>
+    <td>Customers and users who communicate with us directly (e.g. support)</td>
+  </tr>
+  <tr>
+    <td><strong>Payment data</strong></td>
+    <td>
+      <ul>
+        <li>Billing address</li>
+        <li>Customer number</li>
+        <li>Customer history</li>
+        <li>Credit rating information</li>
+      </ul>
+    </td>
+    <td>
+      <p>Customers who use services that require payment</p>
+      <p>Credit rating information: Only customers who pay by invoice</p>
+    </td>
+  </tr>
+  <tr>
+    <td><strong>Data traffic data</strong></td>
+    <td>
+      <ul>
+        <li>User agent</li>
+        <li>IP addresses</li>
+        <li>Operating system</li>
+        <li>Time and date</li>
+        <li>URL</li>
+        <li>Referrer URL</li>
+        <li>Accept Language</li>
+      </ul>
+    </td>
+    <td>All users</td>
+  </tr>  
+</table>
+
+## Scope and responsibility
+
+Under the Agreement, the Processor shall process Personal Data on behalf of the Customer.
+
+This Annex applies to all processing of Customer&#39;s data (including data of the users of Customer&#39;s organization) with reference to persons (&quot; **Personal Data**&quot;) which is related to the Agreement and which is carried out by the Processor, its employees or agents.
```

Comment:
What does the `Customer&#39` has for a purpose, should it not just be  `Customer`?

Reply:
docx to md converter inserted ASCI codes. This should be an apostrophe... I'll make a replace of the ASCI chars.

Reply:
ok